s4e-web/cypress/integration/user-options/user-options-save-view-as-configuration.spec.ts

```diff
@@ -24,30 +24,31 @@ context.skip('Map Views', () => {
       .viewsCountShouldBe(1)
       .deleteNth(0)
       .changeContextTo(ConfirmModal)
-      .acceptAndChangeContextTo(GeneralModal)
-      .closeAndChangeContext(MapViews);
+    GeneralModal
+      .closeModal();
   });
 
   it.only('should display view', () => {
-    Layers
+    MapLayers
       .selectNthSidebarLayer(1)
       .changeContextTo(MapViews)
       .openSaveViewsModal()
       .addView('test-view')
-      .changeContextTo(Layers)
+      .changeContextTo(MapLayers)
       .unselectNthSidebarLayer(1)
       .changeContextTo(MapViews)
       .openViewsModal()
       .viewsCountShouldBe(1)
       .selectNth(0)
-      .changeContextTo(Layers)
+      .changeContextTo(MapLayers)
```

Comment:
We should drop context change idea in functions and work on `MapLayers` directly.

Reply:
this will be change in next PR